src/index.html

```diff
@@ -3,10 +3,332 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="page-wrapper">
+      <header class="header">
+        <a href="#" class="header__logo">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__link-wrapper">
+              <a
+              href="#"
+              class="nav__link nav__link--is-active is-active"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="nav__link-wrapper">
+              <a href="#" class="nav__link">Samsung</a>
+            </li>
+            <li class="nav__link-wrapper">
+              <a href="#" class="nav__link">Смарфоны и телефоны</a>
+            </li>
+            <li class="nav__link-wrapper">
+              <a
+                href="#"
+                class="nav__link"
+                data-qa="nav-hover"
+              >
+                Ноутбуки и компьютеры
+              </a>
+            </li>
+            <li class="nav__link-wrapper">
+              <a href="#" class="nav__link">Гаджеты</a>
+            </li>
+            <li class="nav__link-wrapper">
+              <a href="#" class="nav__link">Планшеты</a>
+            </li>
+            <li class="nav__link-wrapper">
+              <a href="#" class="nav__link">ФОТО</a>
+            </li>
+            <li class="nav__link-wrapper">
+              <a href="#" class="nav__link">ВИДЕО</a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+      <main class="products">
+        <div class="card" data-qa="card">
+          <figure class="card__img-container">
+            <img
+              src="images/imac.jpeg"
+              alt="imac image"
+              class="card__img"
+            >
+            <figcaption class="card__img-description">
+              Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </figcaption>
+          </figure>
+          <p class="card__code">Код товара: <span>195434</span></p>
+          <div class="card__review-container">
+            <div class="card__review-stars">
+              <span class="card__star"></span>
+              <span class="card__star"></span>
+              <span class="card__star"></span>
+              <span class="card__star"></span>
+              <span class="card__star"></span>
+            </div>
```

Comment:
`stars` should be separate block reused from previous task.
`stars--4` modifier should also be used and work correctly.